configure.ac

```diff
@@ -17,6 +17,7 @@ AM_INIT_AUTOMAKE([1.9 foreign tar-ustar -Wall -Wno-portability]
                              *1.11*);;
                              *) echo serial-tests;;
                              esac]))
+AM_MAINTAINER_MODE([enable])
```

Comment:
Note to self: [as it turns out](https://blogs.gnome.org/desrt/2011/09/08/am_maintainer_mode-is-not-cool/), maintainer mode is enabled by default, and `AM_MAINTAINER_MODE([enable])` actually allows us to **disable** it using `--disable-maintainer-mode`. So this does not change the default semantics of ./configure.

Reply:
Yes, that was the intension. I agree with the linked document (as long as the generated files are not committed to Git), but I needed a way to avoid accidental reconfiguration in Docker RO mount.